src/components/LandscapePlot.tsx

```diff
@@ -0,0 +1,186 @@
+import autobind from "autobind-decorator";
+import {CBIOPORTAL_VICTORY_THEME, ScatterPlot} from "cbioportal-frontend-commons";
+import _ from "lodash";
+import {computed} from "mobx";
+import {observer} from "mobx-react";
+import * as React from "react";
+
+import {ITumorTypeFrequencySummary} from "../model/GeneFrequencySummary";
+import GeneFrequencyStore from "../store/GeneFrequencyStore";
+import {
+    dataPointSize,
+    findPathogenicGermlineFrequency,
+    findPercentBiallelic,
+    IScatterPlotDatum
+} from "../util/PlotUtils";
+import {dataPointFill} from "../util/PlotUtils";
+
+import "./LandscapePlot.css";
+
+interface ILandscapePlotProps {
+    frequencyStore: GeneFrequencyStore;
+}
+
+function tumorFrequencyDatumToScatterPlotDatum(d: ITumorTypeFrequencySummary): IScatterPlotDatum {
+    return {
+        x: d.hugoSymbol,
+        y: d.tumorType,
+        datum: d
+    };
+}
+
+function generateTheme() {
+    const theme = _.cloneDeep(CBIOPORTAL_VICTORY_THEME);
+    theme.axis.style.tickLabels.fontSize *= 0.9;
+
+    return theme;
+}
+
+@observer
+class LandscapePlot extends React.Component<ILandscapePlotProps>
+{
+    @computed
+    public get scatterPlotData(): IScatterPlotDatum[]
+    {
+        return this.props.frequencyStore.tumorTypeFrequencySummaryData
+            .map(tumorFrequencyDatumToScatterPlotDatum);
+    }
+
+    @computed
+    public get genesWithSignificantPathogenicGermlineRatio()
+    {
+        return this.props.frequencyStore.geneFrequencySummaryData.filter(d => {
+            const pathogenicGermline = findPathogenicGermlineFrequency(d);
+            const percentBiallelic = findPercentBiallelic(d);
+
+            return (
+                pathogenicGermline && pathogenicGermline.frequency >= 0.0005 &&
```

Comment:
Not sure `0.0005` is the best threshold to filter insignificant genes. Also, we should probably move this to either a constant variable or a component property.

Reply:
yeah this works for now i think 👍 . If it's more a matter of trying to keep the plot decently small we could also do `Top x` or something